src/basekernels/matern.jl

```diff
@@ -77,7 +77,7 @@ end
 
 Matern32Kernel(; metric=Euclidean()) = Matern32Kernel(metric)
 
-kappa(::Matern32Kernel, d::Real) = (1 + sqrt(3) * d) * exp(-sqrt(3) * d)
+kappa(::Matern32Kernel, d::T) where {T<:Real} = (1 + T(sqrt(3)) * d) * exp(-T(sqrt(3)) * d)
```

Comment:
Can't T(sqrt(3)), or at least sqrt(3) be precalculated?

Reply:
One could use
```suggestion
function kappa(::Matern32Kernel, d::Real)
    x = sqrt3 * d
    return (1 + x) * exp(-x)
end
```
and load `IrrationalConstants.sqrt3` in src/KernelFunctions.jl